assignments/prototypes.js

```diff
@@ -16,13 +16,30 @@
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
 
+function GameObject(gameObjectAttributes){
+  this.createdAt = gameObjectAttributes.createdAt;
+  this.dimensions = gameObjectAttributes.dimensions;
+}
+GameObject.prototype.destroy = function () {
+  return `${this.name} was removed from the game`; 
+}
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats(charStatsAttribute){
+  this.hp = charStatsAttribute.hp;
+  this.name = charStatsAttribute.name; 
+  GameObject.call(this, charStatsAttribute); 
```

Comment:
This line should be at the top... I thought it was why some of the console logs returned undefined... 
I'm accepting that it's too late and I don't know. Thank the lord you'll never use prototypes like this again :)